.gitignore

```diff
@@ -8,6 +8,8 @@ build/
 .idea
 # If you're building an application, you may want to check-in your pubspec.lock
 pubspec.lock
+macos/Flutter/
+windows/flutter/
```

Comment:
We don't have to do this, because we may need some other time. But is your branch not stable with dev

Reply:
Okay, I think i'd stop doing "git add ." that adds up all the files